tsconfig.json

```diff
@@ -1,12 +1,14 @@
 {
   "exclude": [
-    "test/*"
+    "test/*",
+    "tasks/*",
```

Comment:
The two lines above can simply be `test` and `tasks`. No need to have the `/*` at the end for everything in the folder to be included.